PMM-12473   Unable to add external service with a query string in the… (#1196)
percona/pmm-doc: docs/release-notes/2.41.1.md

## Bugs Fixed

> - [PMM-12473](https://jira.percona.com/browse/PMM-12473) - Fixed monitoring of external services with query string parameters in metrics path.

Reviewer: ```suggestion
- [PMM-12473](https://jira.percona.com/browse/PMM-12473) - Fixed monitoring of external services when query string parameters are passed to `--metrics-path`.
```
